### DISCUSSION TITLE
您好  firmware-selector.immortalwrt.org  在这里直接下载的X86-64 应该怎么安装DOCKER呢

### DISCUSSION THREAD

**Discussion by tfnhui** [immortalwrt/immortalwrt#1069] [Q&A]
is:issue is:open 您好  firmware-selector.immortalwrt.org  在这里直接下载的X86-64 应该怎么安装DOCKER呢

**Comment by science2468** [comment]
刷好固件后，去imm里面装luci-i18n-dockerman-zh-cn

**Reply by tfnhui** [reply]
![C8JDE3T2A4%X4(FR0{TNNCD](https://github.com/immortalwrt/immortalwrt/assets/95342475/0339655b-e857-4cc4-9b34-79c8e41aaf9e)
您好  提示安装不成功  会提示这个    但是刷新后 上面有了docker了  不过有没有影响呢？

**Reply by science2468** [reply]
等。还有你这空间不够

**Comment by science2468** [comment]
等。然后刷新看看有没有docker

Sent from [Proton Mail](https://proton.me/mail/home) for iOS

2023 年 9 月 15 日星期五 12:08, tfnhui ***@***.***(mailto:2023 年 9 月 15 日星期五 12:08, tfnhui <<a href=)> 来信：

**Reply by tfnhui** [reply]
嗯 有了  现在卡在不会扩容。。。。查老半天了  

还有固件是不是不能 重置系统 呢  恢复配置点不呢
![%%AQH)@%)G}C01@2P{5@HOB](https://github.com/immortalwrt/immortalwrt/assets/95342475/b914f583-c08a-4227-a3df-2dbd68382dc9)

**Reply by science2468** [reply]
去百度、去Google、去恩山。还有ext4类型的固件是没有“执行重置”

**Reply by science2468** [reply]
![image](https://github.com/immortalwrt/immortalwrt/assets/102643084/abc9cc5d-fd50-49d3-8a75-3e5a9539749b)
带“squashfs”才有“执行重置“

**Comment by tfnhui** [comment]
谢谢您啊  都好了 另外 想问一下  自己编译 和下载您的 自己安装需要的软件  两者使用有区别吗

**Reply by science2468** [reply]
一般情况没有区别。具体情况具体分析。
